lxd/instance/drivers/driver_qemu.go

```diff
@@ -8540,6 +8545,10 @@ func (d *qemu) Info() instance.Info {
 		data.Version = "unknown" // Not necessarily an error that should prevent us using driver.
 	}
 
+	if shared.InSnap() && os.Getenv("SNAP_EXT_QEMU_PREFIX") != "" {
+		data.Version = data.Version + " (external)"
```

Comment:
Nice :)

This would be something to check in the tests to using `lxc query /1.0 | jq ...`